oemoflex/preprocessing.py

```diff
@@ -0,0 +1,261 @@
+import logging
+import os
+
+import pandas as pd
+
+
+bus_list = [
+    'AT-el-bus',
+    'BE-el-bus',
+    'CH-el-bus',
+    'CZ-el-bus',
+    'DE-el-bus',
+    'DK-el-bus',
+    'FR-el-bus',
+    'IT-el-bus',
+    'LU-el-bus',
+    'NL-el-bus',
+    'PL-el-bus',
+]
+
+country_list = [
+    'AT',
+    'BE',
+    'CH',
+    'CZ',
+    'DE',
+    'DK',
+    'FR',
+    'IT',
+    'LU',
+    'NL',
+    'PL',
+]
+
+link_list = [
+    'AT-CH',
+    'AT-CZ',
+    'AT-IT',
+    'BE-FR',
+    'BE-LU',
+    'BE-NL',
+    'CH-FR',
+    'CH-IT',
+    'CZ-PL',
+    'DE-AT',
+    'DE-BE',
+    'DE-CH',
+    'DE-CZ',
+    'DE-DK',
+    'DE-FR',
+    'DE-LU',
+    'DE-NL',
+    'DE-PL',
+    'DK-NL',
+    'FR-IT',
+    'FR-LU',
+
+]
+
+datetimeindex = pd.date_range(start='2019-01-01', freq='H', periods=8760)
```

Comment:
Should this be the time index of the scenario year for consistency? Given that all input data is consistent with that year, it makes sense to get that right, in particular for the weekends with their impact on load profiles.

Reply:
I have checked the timeseries in `oemo-flex/experiment_1/001_data_raw/Data/In/v0.02/Energy/FinalEnergy/Electricity/FlexMex1_AT_2050.csv` and aligned it with a 2050 calendar. The load's weekly drops don't align with the weekend. I would like to give some example but I couldn't get a clear plot yet.

Reply:
Because the alignment is different in v0.01 and v0.02 and seems not to be solved in one line of code (it may even need adjustments in the time series themselves), I propose to handle this issue in another PR. Do you agree?

Reply:
I agree. We cannot solve this here.